packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormInput/CoinmarketFormInputCryptoSelect.tsx

```diff
@@ -113,7 +131,7 @@ export const CoinmarketFormInputCryptoSelect = <
                         }}
                         data-testid="@coinmarket/form/select-crypto"
                         isClearable={false}
-                        isSearchable
+                        menuIsOpen={false}
```

Comment:
maybe could you please rename it to`isMenuOpen`? 🙏

Reply:
Nice improvement. I have renamed it, but it is trezor component as well. I'm not sure it is related to this PR. Anyway it is in a separate commit - https://github.com/trezor/trezor-suite/pull/14551/commits/0c30002bc25e903a5f822724631065804f1adc0b